src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -264,6 +219,9 @@ public function escapeValue($value)
         return $value;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function unescapeValue($value)
```

Comment:
After all IMO we should at least keep ``escapeValue`` which may be useful to build parameters:
```php
$parameter = '%foo%'.$parameterBag->escapeValue('%foobar%');